src/components/request/RequestHero.tsx

```diff
@@ -39,7 +46,18 @@ const RequestHero: FC<Props> = ({ chainId }) => {
               </HeroButtonFlex>
             </HeroButton>
 
-            <HeroButton>Request</HeroButton>
+            <HeroButton>
+              {requestState === oracle.types.state.RequestState.Requested ? (
+                "Request"
+              ) : requestState === oracle.types.state.RequestState.Disputed ? (
+                <HeroButtonFlex>
+                  <HeroLogoSmall src={alertIcon} alt="alert_icon" />
+                  <HeroButtonText>Disputed</HeroButtonText>
+                </HeroButtonFlex>
+              ) : (
+                "Proposal"
```

Comment:
im not sure this logic is right, basically if its not disputed or requested its proposed? i think if its waiting on dvm its going to show proposal

Reply:
I think this is the full flow, but im not 100% sure about it so correct me if I'm wrong:
- If its `Requested` then it would mean that the Oracle is waiting on someone to propose a price
- If its `Proposed` it would mean that we are in the OO liveness period
- If its `Disputed` it would mean that someone disputed and we are now waiting on the DVM
- If its `Resolved` it means the DVM / OO liveness period is over and the request is finalized

Reply:
Resolved hides the UI anyway.

Reply:
Adjusted.